app/views/organizers/users_info/index.html.slim

```diff
@@ -9,7 +9,7 @@ h1.header User info
         = role.capitalize
 
     h5 Type
-    - %w(all sponsored voluntary).each do |kind|
+    - %w(all sponsored deprecated_voluntary).each do |kind|
```

Comment:
I'd again suggest to remove the deprecated column here but instead maybe look for a way to include the new type of team "half-time" or however it should be called.